react-search-component/my-app/src/SearchableList.jsx

```diff
@@ -0,0 +1,50 @@
+import { useState } from 'react';
+
+export default function SearchableList({ quotes }) {
+  const [currentText, setCurrentText] = useState('');
+
+  function handleText(e) {
+    setCurrentText(e.target.value);
+  }
+
+  const filter = quotes.filter((qts) =>
+    qts.toLowerCase().includes(currentText.toLowerCase())
+  );
+  return (
+    <div className="row">
+      <div className="column-one-fourth">
+        <SearchBar current={currentText} onChange={handleText} />
+      </div>
+      <div className="column-three-fourth">
+        <ListOfQuotes filter={filter} />
+      </div>
+    </div>
+  );
+}
+
+function SearchBar({ current, onChange }) {
+  return (
+    <>
+      <input
+        type="text"
+        placeholder="search"
+        value={current}
+        onChange={onChange}
+      />
+    </>
+  );
+}
+
+function ListOfQuotes({ filter }) {
+  return (
+    <>
+      <ul>
+        {filter.length === 0 ? (
+          <p>No items match the filter.</p>
+        ) : (
+          filter.map((qts, i) => <li key={i}>{qts}</li>)
```

Comment:
Personal preference, but using `qts` in `line 10` makes sense bc you already have `quotes` defined in scope. 

Here however, on `line 45`, I would just use `quotes` so its even more clear! It's a bit obvious already because of the component name of `ListOfQuotes`, but this makes it even clearer! 

This is kind of a person to person preference! Just a friendly suggestion on clearer variable naming! If someone else had to fix a bug in the future, you've made it a tinier bit easier for them to figure out your code!